picard/ui/sortablecheckboxlist.py

```diff
@@ -0,0 +1,171 @@
+from PyQt4 import QtGui
+from PyQt4 import QtCore
+import sys
+from functools import partial
+
+
+class Signal:
+
+    """Signal class from http://blog.abstractfactory.io/dynamic-signals-in-pyqt/
```

Comment:
We could i guess, but i didn't want to bother with them.
I don't like pyqtsignals mostly because of the "typed" arguments.
Frankly, this little class does the job perfectly, is simpler to use and more pythonic.
Though it should be somewhere in util/